components/Partner/index.vue

```diff
@@ -153,6 +153,7 @@ export default {
     } else {
       this.partners = data
     }
+    console.log(this.partners)
```

Comment:
## Minor

I think it's unnecessary code 👍🏽

Reply:
whoops, sleepy code 🤣 
thankss, fixed [here](https://github.com/jabardigitalservice/dedi-landing-page/pull/157/commits/1d0c87a8994fe971bbfd0636b5cbe96a74c5fca6)